lcd/pyb_i2c_lcd_test.py

```diff
@@ -1,7 +1,7 @@
 """Implements a HD44780 character LCD connected via PCF8574 on I2C."""
 
 from pyb import I2C, delay, millis
-from lcd_api import LcdApi
+from pyb_i2c_lcd import I2cLcd
```

Comment:
For some reason, I wasn't seeing all of the commits, so my comments might not be relevant. Let me review everything as a whole.